zqd/space/endpoint_test.go

```diff
@@ -0,0 +1,46 @@
+package space_test
```

Comment:
I don't see a reason for this to be a different package from the other files in this directory.

Reply:
See answer above.